src/fillTank.test.js

```diff
@@ -1,11 +1,130 @@
+/* eslint-disable max-len */
 'use strict';
 
 describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+  const { fillTank } = require('./fillTank');
 
-  it('should ', () => {
+  it('Should be called', () => {
+    expect(fillTank).toBeInstanceOf(Function);
+  });
+
+  it('Should tank to max capacity, if customer want to much ', () => {
+    const customer = {
+      money: 2085,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 35,
+      },
+    };
+
+    fillTank(customer, 28, 15);
+
+    expect(customer).toEqual({
+      money: 1945,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 40,
+      },
+    });
+  });
+
+  it('Should tank to max capacity, if customer doesn\'t say amount', () => {
```

Comment:
The test description 'Should tank to max capacity, if customer doesn\'t say amount' is slightly unclear. Consider rephrasing to 'Should fill to max capacity if amount is not specified'.